src/i18n/en.js

```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    all_of_plan: `Everything from {plan} plus:`,
     calculate_price: 'Calculate Price',
+    cancel_campaign: 'Cancel Campaign',
+    compare_features: 'Comparare Features',
     description: 'Do you send periodically or one newsletter per month?',
+    email_parameter: 'Email parameter',
+    features: 'Features:',
     per_month: 'per month',
     recommended: 'Recommended',
     selected_type: 'Plans selected by: ',
+    shipping_limit: 'Shipping limit',
     since: 'Since',
+    site_tracking: 'Site tracking',
+    smart_campaigns: 'Smart Campaigns',
```

Comment:
```suggestion
    smart_campaigns: 'Smart Send Out',
```
![image](https://user-images.githubusercontent.com/2439363/91483033-6918a400-e87d-11ea-8e97-65f1d62d86ec.png)